label_maker/package.py

```diff
@@ -73,25 +74,37 @@ def package_directory(dest_folder, classes, imagery, ml_type, seed=False,
 
     # open the images and load those plus the labels into the final arrays
     if is_tif(imagery):  # if a TIF is provided, use jpg as tile format
-        image_format = '.jpg'
+        img_dtype = rasterio.open(imagery).profile['dtype']
+        try:
+            # MS bands written to png
+            kwargs['band_indicies']
+            image_format = '.png'
+        except KeyError:
+            # 3 band RGB written to jpg
+            image_format = '.jpg'
 
     else:
+        img_dtype = np.uint8
         image_format = get_image_format(imagery, kwargs)
 
     for tile in tiles:
         image_file = op.join(dest_folder, 'tiles', '{}{}'.format(tile, image_format))
         try:
-            img = Image.open(image_file)
+            #img = Image.open(image_file)
+            img = rasterio.open(image_file)
```

Comment:
If we use the PIL `.open` we won't have to rollaxis later

Reply:
PIL `.open` added extra dimensions to the 2 band image array and made it 4 dimensions, so I switched to rasterio

Reply:
I think image drivers will interpret that as greyscale + alpha